### DISCUSSION TITLE
[Kavita] Kobo reader sync support

### DISCUSSION THREAD

**Discussion by DieselTech** [Kareadita/Kavita#2538] [Ideas]
### Idea Description

Posted by runtime
· Jul 2022
· 35 Upvotes
https://feats.kavitareader.com/posts/171/kobo-reader-sync-support

Description

Add Kobo reader sync integration like calibre-web has https://github.com/janeczku/calibre-web/wiki/Kobo-Integration

### Duration of Using Kavita

42

### Idea Category

Feature Enhancement

### Agreement

- [X] I agree that this is solely for submitting ideas, and I will search for existing ideas before posting.

**Comment by ch13696** [comment]
I second this feature request. Komga has it added in their system if you want to see how exactly it works.

https://komga.org/docs/guides/kobo/

I don't know how simple it is to add, but it is a very useful feature for syncing progress and I would much rather use it with Kavita because of the reading statistics.

**Comment by Dyad-ITSS** [comment]
Another second from my side, only feature missing for me to move over.

**Comment by cr0ssmind** [comment]
+1, please implement it

**Comment by rodrigoGA** [comment]
Is there any news on this? Is it planned?

**Reply by majora2007** [reply]
The label says accepted, so yes it's planned.

**Comment by majora2007** [comment]
Just FYI, while the full blown Kobo reader sync support is not yet delivered, Koreader sync support was build and is coming with v0.8.7 (releasing in July). Hopefully this can tide you over. 

Read about it here: 
https://wiki.kavitareader.com/guides/3rdparty/koreader/#koreader

**Reply by Huguesmmm** [reply]
Can I help with the Kobo sync? Is there already someone on it?

**Reply by majora2007** [reply]
I'm open to a plan of approach with documentation to review before you get started, if you want to help out. A PR without approval on approach would be unwise. 

I haven't yet looked into this, but I do want to ensure a killer experience that matches or beats the experience of other reading software.

This is still one of my priorities of the year, however, since the Annotation system took 3+ months (mainly due to one bug), this might get pushed to early next year.

**Comment by Beanosss** [comment]
+1 this would be fantastic!

**Comment by ydainna** [comment]
Hi, any news about synchronization with Kobo? Thank you.

**Reply by majora2007** [reply]
There is no update at this time. User above never reached out post contact. Our hands have been full trying to accomplish core goals for the platform. 

There was a Koreader plugin however a user just launched that uses the official API rather than OPDS (so much better progress sync):
https://github.com/fpammer/kamare.koplugin

**Comment by majora2007** [comment]
Hi All,

Just to let you know, this is planned for v0.9.2 (the release after next), unless something major comes up.
